.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "0.10"
+  - "6.3.1"
```

Comment:
Good question. Since I still have no access to that machine, I can't help to verify.